arch/xtensa/src/common/xtensa_openeth.c

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * arch/xtensa/src/esp32/esp32_openeth.c
+ * arch/xtensa/src/common/xtensa_openeth.c
```

Comment:
can this driver be used by other non esp chip?

Reply:
No. AFAIK this is only for esp32 family on qemu.

Another chip with a real openeth peripheral will include a more complete driver and features that are not available in the openeth qemu implementation, which has many parts mocked.

Maybe this can be used by other qemu like arm or riscv, but I have not tested as in those cases, network can usually be done with virtio or some other better interface (than mocking openeth nic).

Reply:
common folder is for all chips which are based on xtensa architecture, not only for esp32, since openeth can only work for esp32, it's better to move to the esp32 common folder.

Reply:
I think you need create a new folder for esp common stuff, @tmedicci do you have some suggestion?

Reply:
done.